book/src/external-attacher.md

```diff
@@ -8,6 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-attacher v3.0.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.0.0) | [release-3.0](https://github.com/kubernetes-csi/external-attacher/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.0.0 | v1.17 | - | v1.17
```

Comment:
@jsafrane csi-attacher release page says recommended 1.18 for v3.0.0 and v2.2.0, but it's 1.17 here. Which one is correct?

Reply:
I think it's 1.17, I remember fixing README in v3.0.0 to 1.17